src/CommunityToolkit.Aspire.Hosting.OpenTelemetryCollector/OpenTelemetryCollectorSettings.cs

```diff
@@ -0,0 +1,27 @@
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Settings for the OpenTelemetry Collector
+/// </summary>
+public class OpenTelemetryCollectorSettings
+{
+    /// <summary>
+    /// The version of the collector, defaults to latest
+    /// </summary>
+    public string CollectorVersion { get; set; } = "latest";
```

Comment:
We should pin to a version so it's known stable per the package version

Reply:
I'm not convinced that pinning is the right approach for that, especially since there isn't a v1 yet. It's generally accepted that people don't pin even in production.